example/App.js

```diff
@@ -1,41 +1,51 @@
-import React from "react";
-import { StyleSheet, Text, View, SafeAreaView } from "react-native";
+import React, { useState } from "react";
+import { 
+  Platform,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  useColorScheme,
+  View,
+} from "react-native";
 
 import EmojiSelector, { Categories } from "./module";
 const THEME = "#007AFF";
 
-export default class App extends React.Component {
-  state = {
-    emoji: " "
-  };
-  render() {
-    return (
-      <SafeAreaView style={styles.container}>
-        <Text>Please select the emoji you would like to use</Text>
-        <View style={styles.display}>
-          <Text style={{ fontSize: 64, backgroundColor: "transparent" }}>
-            {this.state.emoji}
-          </Text>
-        </View>
-        <EmojiSelector
-          onEmojiSelected={emoji => this.setState({ emoji })}
-          showSearchBar={true}
-          showTabs={true}
-          showHistory={true}
-          showSectionTitles={true}
-          category={Categories.all}
-        />
-      </SafeAreaView>
-    );
-  }
+export default function App() {
+  const [emoji, setEmoji] = useState(" ");
+  const isDarkMode = useColorScheme() === 'dark';
```

Comment:
we should use app provided dark mode setting in case user choose to set it manually

Reply:
sure, make sense!

Reply:
Should dark-mode be set manually, `platformcolor` should not be used due to possible UI issues. For example, a manually set dark-mode with a light system theme, will display a lightly coloured search bar against a white background resulting in low contrast.

Setting the styles manually will overcome this limitation